neanderthal/src/main/kotlin/au/com/outware/neanderthal/data/repository/VariantSharedPreferencesRepository.kt

```diff
@@ -81,6 +90,17 @@ class VariantSharedPreferencesRepository(val klass: Class<out Any>,
         return variants
     }
 
+    override fun getDefaultVariants(): List<Variant> {
```

Comment:
you could probably take this one step further by remove the suffix here, so return 
`Variant(name.removeSuffix(VARIANT_DEFAULT), gson.fromJson(sharedPreferences.getString(name, null), klass))`